### PR TITLE
RATIS-1318. Remove incubating

### PR DESCRIPTION
**Pull request by runzhiwang**
## What changes were proposed in this pull request?

Remove incubating

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/RATIS-1318

## How was this patch tested?

no need new ut.

### CODE REVIEW

**Comment by szetszwo** [comment]
@runzhiwang , thanks a lot for working on this.  There are a few other "incubator" or "incubating" found in make_rc.sh.
{code}
-- ./dev-support/make_rc.sh ---------------------------------------------------
 82:   svn co https://dist.apache.org/repos/dist/dev/incubator/ratis "$SVNDISTDIR"
119:   tar zvxf "$projectdir/ratis-assembly/target/apache-ratis-incubating-${RATISVERSION}-src.tar.gz"
120:   cd "apache-ratis-incubating-${RATISVERSION}"
130:   cp "$WORKINGDIR/apache-ratis-incubating-${RATISVERSION}/ratis-assembly/target/apache-ratis-incubating-${RATISVERSION}-bin.tar.gz" "apache-ratis-incubating-${RATISVERSION}-bin.tar.gz"
131:   cp "$projectdir/ratis-assembly/target/apache-ratis-incubating-${RATISVERSION}-src.tar.gz" "apache-ratis-incubating-${RATISVERSION}-src.tar.gz"
182:    10. (If the vote passed): Send out the next voting mail to the incubator mailing list.
{code}

File: dev-support/vagrant/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 -->
 # What is this?
 
-This is a series of scripts for [Vagrant](https://vagrantup.com) to stand-up and test [Apache (Incubating) Ratis](https://ratis.incubator.apache.org/) servers and clients. One needs to have Vagrant and [VirtualBox](https://virtualbox.org) installed to stand-up these tests environments.
+This is a series of scripts for [Vagrant](https://vagrantup.com) to stand-up and test [Apache Ratis](https://ratis.incubator.apache.org/) servers and clients. One needs to have Vagrant and [VirtualBox](https://virtualbox.org) installed to stand-up these tests environments.
```
Comment (szetszwo) [inline dev-support/vagrant/README.md:16]: There are quite many other changes required in vagrant.  Let's change them all together in a separated JIRA?
```
-- ./dev-support/vagrant/README.md --------------------------------------------
 16: This is a series of scripts for [Vagrant](https://vagrantup.com) to stand-up and test [Apache (Incubating) Ratis](https://ratis.incubator.apache.org/) servers and clients. One needs to have Vagrant and [VirtualBox](https://virtualbox.org) installed to stand-up these tests environments.
   1 occurrence(s)
-- ./dev-support/vagrant/Vagrantfile ------------------------------------------
 24: RATIS_PATH = ::File.join(VAGRANT_HOME, 'incubator-ratis')
   1 occurrence(s)
-- ./dev-support/vagrant/bin/start_ratis_load_gen.sh --------------------------
 22: cd ${HOME}/incubator-ratis
   1 occurrence(s)
-- ./dev-support/vagrant/bin/start_ratis_server.sh ----------------------------
 25: cd ${HOME}/incubator-ratis/
   1 occurrence(s)
-- ./dev-support/vagrant/screenrcs/namazu_hdd_screenrc ------------------------
 23: screen -t Disk0 0 /home/vagrant/namazu/bin/nmz inspectors fs -original-dir /home/vagrant/test_data/data0 -mount-point /home/vagrant/test_data/data0_slowed/ -autopilot /home/vagrant/incubator-ratis/dev-support/vagrant/namazu_configs/hdd_config.toml
 24: screen -t Disk1 0 /home/vagrant/namazu/bin/nmz inspectors fs -original-dir /home/vagrant/test_data/data1 -mount-point /home/vagrant/test_data/data1_slowed/ -autopilot /home/vagrant/incubator-ratis/dev-support/vagrant/namazu_configs/hdd_config.toml
 25: screen -t Disk2 0 /home/vagrant/namazu/bin/nmz inspectors fs -original-dir /home/vagrant/test_data/data2 -mount-point /home/vagrant/test_data/data2_slowed/ -autopilot /home/vagrant/incubator-ratis/dev-support/vagrant/namazu_configs/hdd_config.toml
   3 occurrence(s)
-- ./dev-support/vagrant/screenrcs/ratis_ratis-hdd-slowdown_screenrc ----------
 23: screen -t Server0 0 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data0_slowed n0 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 24: screen -t Server1 1 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data1_slowed n1 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 25: screen -t Server2 2 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data2_slowed n2 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 26: screen -t LoadGen 3 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_load_gen.sh n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
   4 occurrence(s)
-- ./dev-support/vagrant/screenrcs/ratis_ratis-server_screenrc ----------------
 23: screen -t Server0 0 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data0 n0 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 24: screen -t Server1 1 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data1 n1 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 25: screen -t Server2 2 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_server.sh /home/vagrant/test_data/data2 n2 n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
 26: screen -t LoadGen 3 /home/vagrant/incubator-ratis/dev-support/vagrant/bin/start_ratis_load_gen.sh n0:127.0.0.1:6000,n1:127.0.0.1:6001,n2:127.0.0.1:6002
   4 occurrence(s)
```

**Comment by runzhiwang** [comment]
```
-- ./dev-support/make_rc.sh ---------------------------------------------------
 82:   svn co https://dist.apache.org/repos/dist/dev/incubator/ratis "$SVNDISTDIR"
119:   tar zvxf "$projectdir/ratis-assembly/target/apache-ratis-incubating-${RATISVERSION}-src.tar.gz"
120:   cd "apache-ratis-incubating-${RATISVERSION}"
130:   cp "$WORKINGDIR/apache-ratis-incubating-${RATISVERSION}/ratis-assembly/target/apache-ratis-incubating-${RATISVERSION}-bin.tar.gz" "apache-ratis-incubating-${RATISVERSION}-bin.tar.gz"
131:   cp "$projectdir/ratis-assembly/target/apache-ratis-incubating-${RATISVERSION}-src.tar.gz" "apache-ratis-incubating-${RATISVERSION}-src.tar.gz"
182:    10. (If the vote passed): Send out the next voting mail to the incubator mailing list.
```

@szetszwo  Hi, line 119, 120, 130, 131 already changed in this PR. line 82 need to change dir, if we remove incubator, the dir will be https://dist.apache.org/repos/dist/dev/ratis, which does not exist.

File: dev-support/vagrant/README.md (hunk 1 above)
Comment (runzhiwang) [reply dev-support/vagrant/README.md:16]: Sure

**Comment by szetszwo** [comment]
@runzhiwang , I just have created https://dist.apache.org/repos/dist/dev/ratis/  and copied KEYS to there.

**Comment by runzhiwang** [comment]
>  I just have created https://dist.apache.org/repos/dist/dev/ratis/ and copied KEYS to there.

@szetszwo Thanks, do we put release 2.0.0 to https://dist.apache.org/repos/dist/dev/ratis/? and do we need to copy the previous release such as 1.0.0 to https://dist.apache.org/repos/dist/dev/ratis/?

**Comment by szetszwo** [comment]
Yes, we should put the 2.0.0 release to the new dir.  For the old releases, we will move them once we have fixed the links in the web pages.

**Comment by runzhiwang** [comment]
@szetszwo Hi, I have updated the patch, there are other "incubator" and "incubating" related to repository name and email, so I did not change them.

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Pull request merged**